src/why3prover.ml

```diff
@@ -0,0 +1,78 @@
+(** Calling provers in Why3 *)
+
+open Timed
+
+(** [!current_prover] contains the name of the current prover (can be changed
+    by using the set prover <name> command). *)
+let current_prover : string ref = ref "Alt-Ergo"
+
+(** [time_limit] is the time limit of a prover while finding a proof. *)
```

Comment:
Add time unit.

Reply:
Added in 237accc649981c1ca3e4a92a746acd288891bfb1